qemu/qemu.go

```diff
@@ -85,6 +86,9 @@ const (
 
 	// VHostVSockPCI is the vhost vsock pci driver.
 	VHostVSockPCI DeviceDriver = "vhost-vsock-pci"
+
+	// VHostVSockCCW is the vhost vsock ccw driver.
+	VHostVSockCCW DeviceDriver = "vhost-vsock-ccw"
```

Comment:
Rather than add -ccw version add a generic version.

Reply:
for vhost-vsock generic version does not exist.